incubator-tools/label_migration_child_to_parent/label_migration_child_to_parent.ipynb

```diff
@@ -0,0 +1,260 @@
+{
```

Comment:
<p>This notebook seems way too specific for this one use case. It should probably be restructured to work with any Child to Parent label migration</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>